src/test/java/org/folio/rest/impl/ExportManagerTest.java

```diff
@@ -55,25 +55,25 @@ void shouldReturn_422Status_ifRequestIsWrong(VertxTestContext context) {
       .post(EXPORT_URL);
     // then
     context.verify(() -> {
-      assertEquals(HttpStatus.SC_UNPROCESSABLE_ENTITY, response.getStatusCode());
+      assertEquals(HttpStatus.SC_BAD_REQUEST, response.getStatusCode());
       context.completeNow();
     });
   }
+
   @Test
-  void shouldReturn_400Status_ifJobProfileNotFound(VertxTestContext context) {
+  void shouldReturn_400Status_ifFileDefinitionNotFound(VertxTestContext context) {
     // given
     ExportRequest exportRequest = new ExportRequest()
       .withFileDefinitionId(UUID.randomUUID().toString())
-      .withJobProfileId(UUID.randomUUID().toString());
+      .withJobProfileId(DEFAULT_JOB_PROFILE_ID);
     // when
     Response response = RestAssured.given()
       .spec(jsonRequestSpecification)
-      .body(JsonObject.mapFrom(exportRequest)
-        .encode())
+      .body(JsonObject.mapFrom(exportRequest).encode())
       .when()
       .post(EXPORT_URL);
     // then
-    context.verify(() -> {
+    context.verify(()->{
```

Comment:
Can you validate the message as well please, otherwise we actually cannot differentiate between the above test and this one